### ISSUE TITLE
Fatal error in Redis cache

### ISSUE THREAD

**Issue by matsduf** [zonemaster/zonemaster-engine#1326]
I followed the instructions in "[Configuration: Global cache in Zonemaster-Engine]" (develop branch) on Ubuntu 22.04 and created and updated profile where the following has been added:
```
$ head /etc/zonemaster/profile.json
{
    "cache": {
        "redis": {
            "server": "127.0.0.1:6379",
            "expire": 300
        }
    },
    "asn_db" : {
        "style" : "Cymru",
        "sources" : {
```

The Redis daemon was running and I ran zonemaster-cli on zone sunet.se with no specific errors (not using the custom profile). I ran it again with the custom profiile:
```
$ zonemaster-cli sunet.se --profile /etc/zonemaster/profile.json
Loading profile from /etc/zonemaster/profile.json.

Seconds Level    Message
======= ======== =======
(...)
   2.53 CRITICAL Fatal error in Zonemaster::Engine::Test::Nameserver: [set] ERR invalid expire time in set,  at /usr/local/share/perl/5.34.0/Zonemaster/Engine/Nameserver/Cache/RedisCache.pm line 96.
(...)
```
I turned the Redis daemon off and got a different errror
```
$ zonemaster-cli sunet.se --profile /etc/zonemaster/profile.json
Loading profile from /etc/zonemaster/profile.json.

Seconds Level    Message
======= ======== =======
Looks OK.
Could not connect to Redis server at 127.0.0.1:6379: Connection refused at /usr/local/share/perl/5.34.0/Zonemaster/Engine/Nameserver/Cache/RedisCache.pm line 50.
```

[Configuration: Global cache in Zonemaster-Engine]: https://github.com/zonemaster/zonemaster/blob/develop/docs/public/configuration/global-cache.md

**Comment by matsduf** [comment]
The error is triggered by TTL = 0, such as the following response:
```
;; ANSWER SECTION:
iis.se.			0 IN CDS 51298 13 2 (
				60A48DB6C1F4B993E3D7C0869C0C535A70C9A6D1899D
				E86563D485B515EE1918 )
```

**Comment by matsduf** [comment]
Redis requires the cache time to be greater than 0.

**Comment by matsduf** [comment]
Another example:

```
$ zonemaster-cli iis.se --profile /etc/zonemaster/profile.json
Loading profile from /etc/zonemaster/profile.json.

Seconds Level    Message
======= ======== =======
   9.45 CRITICAL Fatal error in Zonemaster::Engine::Test::DNSSEC: [set] ERR invalid expire time in set,  at /usr/local/share/perl/5.34.0/Zonemaster/Engine/Nameserver/Cache/RedisCache.pm line 97.
```

**Comment by matsduf** [comment]
A solution is found in #1327.

**Comment by matsduf** [comment]
Resolved by #1327
